masonry_core/src/widgets/button.rs

```diff
@@ -219,16 +220,27 @@ impl Widget for Button {
         let border_width = props.get::<BorderWidth>().unwrap_or(&DEFAULT_BORDER_WIDTH);
         let border_radius = props.get::<CornerRadius>().unwrap_or(&DEFAULT_BORDER_RADII);
 
+        // TODO - Add DEFAULT_BACKGROUND_GRADIENT constant.
```

Comment:
Why? What does that constant achieve/why isn't it in this PR

Reply:
Consistency with the other properties.

I agree it's not super important.